help_to_heat/frontdoor/schemas.py

```diff
@@ -54,6 +57,9 @@
     "applications-closed": {"prev": "supplier", "next": None},
     "application-closed-utility-warehouse": {"prev": "supplier", "next": None},
     "benefits": {"prev": "council-tax-band", "next": "household-income"},
+    "park-home": {"prev": "own-property", "next": "address"},
+    "park-home-main-residence": {"prev": "park-home", "next": "address"},
+    "park-home-application-closed": {"prev": "park-home-main-residence", "next": "none"},
```

Comment:
I think that should be `None`, not `"none"`.

Reply:
You're absolutely right 😿